### PR TITLE
fixes lack of documentation

### LINKED ISSUE #40: Add code documentation
Even if the functions are pretty self explanatory. It would be nice to actually have a small documentation above each one.

If that could be done before the release of the V4 that would be top though.

### PR DESCRIPTION
**Pull request by ildyria**
fixes #40

### CODE REVIEW

File: app/Http/Controllers/PhotoController.php [hunk 1]
```diff
@@ -201,6 +213,10 @@ public function setDescription(Request $request)
 	}
 
 	/**
+	 * Define if a photo is public.
+	 * We do not advise the use of this and would rather see people use albums visibility
+	 * This would highly simplify the code if we remove this. Do we really want to keep it ?
+	 *
```
Comment (kamil4) [inline app/Http/Controllers/PhotoController.php]: One case where you can't use album visibility is for Unsorted. But the whole Unsorted always struck me as a bit of a strange feature. If we want to support pictures being displayed directly at the top level (without putting them in albums) then that's fine, but that's not what Unsorted does. When you upload a picture at the top level (albums view) it ends up in Unsorted. When you make it public, it also shows up in the Public smart album, but it's still not visible at the top level. What's the use case here? No idea...

File: app/Http/Controllers/PhotoController.php [hunk 2]
```diff
@@ -440,6 +467,8 @@ public function duplicate(Request $request)
 	}
 
 	/**
+	 * Return a photo as an archive just to annoy @SerenaButler.
```
Comment (kamil4) [inline app/Http/Controllers/PhotoController.php]: LOL :smiley:. I actually I have it on my TODO list to fix it (I will comment in https://github.com/LycheeOrg/Lychee-front/issues/119).

File: app/Http/Controllers/SettingsController.php [hunk 3]
```diff
@@ -140,6 +215,14 @@ public function setPublicSearch(Request $request)
 		return (Configs::set('public_search', '0')) ? 'true' : 'false';
 	}
 
+	/**
+	 * Show image overlay by default or not
+	 * (white text in the bottom right corner).
```
Comment (kamil4) [inline app/Http/Controllers/SettingsController.php]: Bottom _left_, I believe?

File: app/Http/Controllers/PhotoController.php (hunk 1 above)
Comment (ildyria) [reply app/Http/Controllers/PhotoController.php]: use case is being able to share a single picture in a private album.
You can then access the picture with the view.php (in Lychee v3) and a similar system in the v4.

File: app/Http/Controllers/SettingsController.php (hunk 3 above)
Comment (ildyria) [reply app/Http/Controllers/SettingsController.php]: yes, the other right.

File: app/Http/Controllers/PhotoController.php (hunk 1 above)
Comment (kamil4) [reply app/Http/Controllers/PhotoController.php]: Yeah, I know. I guess what I was trying to say is that I would find the existing use case a lot more compelling if it applied to top-level images as well -- but there's currently no such thing as top-level images...

**Pull request merged**